docs/examples/daterange/errorMessaging.js

```diff
@@ -0,0 +1,63 @@
+// @flow strict
+import { type Node, useState } from 'react';
+import { DateRange } from 'gestalt-datepicker';
+
+export default function Example(): Node {
+  const [startDate, setStartDate] = useState<Date | null>(null);
```

Comment:
Can we initialize with an invalid date so the error appears by default

Reply:
DONE